README.md

```diff
@@ -26,27 +26,34 @@ Usage
 
 ## `seek(iter, where)`
 
-The `iter` argument must be a `NodeIterator`
-([docs](https://developer.mozilla.org/en-US/docs/Web/API/NodeIterator))
-instance with a `whatToShow` property equal to `NodeFilter.SHOW_TEXT`.
-
-Use the `dom-node-iterator` module for a portable `NodeIterator` polyfill if
-targeting browsers that lack a full implementation that includes the
-`referenceNode` and `pointerBeforeReferenceNode` properties.
-
-The `where` argument is an integer, else an `Element` or `Text` node.
+Adjust the position of a [`NodeIterator`].
 
 If the argument is an integer, seeks the iterator forward (if `where` is
-positive) or backward (if `where` is negative) until `where` characters have
-been traversed or the traversal ends. The iterator may be left with its pointer
-either before or after the reference node.
+positive) or backward (if `where` is negative) until `where` text code units
+have been traversed or the iterator is exhausted.
 
 If the argument is a node, seeks the iterator forward or backward until its
-pointer is positioned immediately before the given node.
+pointer is positioned immediately before the target node.
+
+After seeking, the iterator will refer to the point before its reference node
+(its `pointerBeforeReferencNode` property will be `true`).
 
 Returns the change in the offset. Note that this will be negative when the
 traversal causes the iterator to move backward.
 
+Raises `InvalidStateError` when the root node of the iterator contains no text
+or `iter` is not a `NodeIterator` with a `whatToShow` property equal to
+`NodeFilter.SHOW_TEXT`.
```

Comment:
Not sure if both of these should be `InvalidStateError` or if the first issue is more `InvalidNodeTypeError`. I went with `InvalidStateError` for now thinking that the node type isn't invalid, but it's in an invalid state (has no text).